lib/jsdom/living/nodes/GlobalEventHandlers-impl.js

```diff
@@ -22,14 +22,14 @@ const events = new Set([
   "mousedown", "mouseenter",
   "mouseleave", "mousemove",
   "mouseout", "mouseover",
-  "mouseup", "wheel",
+  "mouseup", "wheel", "mousewheel",
   "pause", "play",
   "playing", "progress",
   "ratechange", "reset",
   "resize", "scroll",
   "securitypolicyviolation",
   "seeked", "seeking",
-  "select", "sort", "stalled",
+  "select", "show", "sort", "stalled",
```

Comment:
These two additions don't seem to be in the [spec](https://html.spec.whatwg.org/multipage/webappapis.html#globaleventhandlers) anymore. See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/GlobalEventHandlers.onmousewheel) for `mousewheel`.

Reply:
but they are still on the test from https://github.com/w3c/web-platform-tests/blob/master/html/browsers/the-window-object/window-properties.html, should I ignore for our case and probably later do a contribution there to remove these 2 additions?

Reply:
Yeah that works.